pyproject.toml

```diff
@@ -0,0 +1,95 @@
+[tool.poetry]
+name = "kiwi_boxed_plugin"
+version = "0.2.45"
+description = "KIWI - Boxed Build Plugin"
+license = "GPL-3.0-or-later"
+readme = "README.rst"
+homepage = "https://osinside.github.io/kiwi/"
+documentation = "https://osinside.github.io/kiwi/plugins/self_contained.html"
+repository = "https://github.com/OSInside/kiwi-boxed-plugin"
+
+authors = [
+   "Marcus Schäfer <marcus.schaefer@suse.com>",
+]
+maintainers = [
+   "Marcus Schäfer <marcus.schaefer@suse.com>",
+   "David Cassany Viladomat <dcassany@suse.com>",
+   "Neal Gompa <neal@gompa.dev>",
+]
+
+packages = [
+   { include = "kiwi_boxed_plugin"},
+]
+
+include = [
+   { path = ".bumpversion.cfg", format = "sdist" },
+   { path = ".coverage*", format = "sdist" },
+   { path = "setup.cfg", format = "sdist" },
+   { path = "doc/source", format = "sdist" },
+   { path = "doc/Makefile", format = "sdist" },
+   { path = "helper", format = "sdist" },
+   { path = "Makefile", format = "sdist" },
+   { path = "package", format = "sdist" },
+   { path = "test", format = "sdist" },
+]
+
+classifiers = [
+   # classifier: http://pypi.python.org/pypi?%3Aaction=list_classifiers
+   "Development Status :: 5 - Production/Stable",
+   "Intended Audience :: Developers",
+   "Operating System :: POSIX :: Linux",
+   "Topic :: System :: Operating System",
+]
+
+[tool.poetry.urls]
+"Bug Tracker" = "https://github.com/OSInside/kiwi-boxed-plugin/issues"
+
+[tool.poetry.dependencies]
+python = "^3.9"
+kiwi = ">=9.21.21"
+docopt-ng = ">=0.9.0"
+requests = ">=2.25.0"
+PyYAML = ">=5.4.0"
+Cerberus = ">=1.3.5"
+progressbar2 = ">=4.2.0"
+setuptools = ">=50"
+
+[tool.poetry.plugins]
+[tool.poetry.plugins."kiwi.tasks"]
+system_boxbuild = "kiwi_boxed_plugin.tasks.system_boxbuild"
+
+[tool.poetry.group.test]
+[tool.poetry.group.test.dependencies]
+# python unit testing framework
+pytest = ">=6.2.0"
+pytest-cov = "*"
+pytest-xdist = "*"
+# Optional dependencies for shell functions integration tests
+pytest-container = {version = "*", optional = true}
+# type checking
+mypy = ">=0.971"
+types-requests = "*"
+types-PyYAML = "*"
+types-mock = "*"
```

Comment:
This isn't needed when using `unittest.mock`

Reply:
yep, deleted